core/src/commands/cloud/secrets/secrets-create.ts

```diff
@@ -86,20 +87,20 @@ export class SecretsCreateCommand extends Command<Args, Opts> {
 
     const cmdLog = log.createLog({ name: "secrets-command" })
 
-    const secrets = (
-      await readInputKeyValueResources({
-        resourceFilePath: secretsFilePath,
-        resourcesFromArgs: args.secrets,
-        resourceName: "secret",
-        log: cmdLog,
-      })
-    ).map(([key, value]) => ({ name: key, value }))
+    const inputSecrets = await readInputKeyValueResources({
+      resourceFilePath: secretsFilePath,
+      resourcesFromArgs: args.secrets,
+      resourceName: "secret",
+      log: cmdLog,
+    })
 
     const api = garden.cloudApi
     if (!api) {
       throw new ConfigurationError({ message: noApiMsg("create", "secrets") })
     }
 
+    const secretsToCreate: Secret[] = inputSecrets.map(([key, value]): Secret => ({ name: key, value }))
```

Comment:
The outer type will be inferred automatically, if you want the code to be less verbose. But it's fine for me like this too 👍 
```suggestion
    const secretsToCreate = inputSecrets.map(([key, value]): Secret => ({ name: key, value }))
```

Reply:
I left it intentionally here and in the rest 2 places. Just to ensure the type-safety :) Is for some reason the inner type is removed, the outer guard will still fail if the mapper returns an incompatible value :D

Reply:
A little excessive for my taste to repeat the type in the same one-liner, but both variants are acceptable for me :)